Description: Install libraries in the default libdir folder
 Do not postfix 64 in install libdir
Author: arnebe <arne@alamut.de> 
Forwarded: no
Last-Update: 2012-01-27

--- a/cmake_modules/FindOS.cmake
+++ b/cmake_modules/FindOS.cmake
@@ -46,14 +46,3 @@
 
 MESSAGE (STATUS "Got System Processor ${CMAKE_SYSTEM_PROCESSOR}")
 
-# 32 or 64 bit Linux
-IF (PROJECT_OS_LINUX)
-    # Set the library directory suffix accordingly
-    IF (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
-        SET (PROJECT_PROC_64BIT TRUE BOOL INTERNAL)
-        MESSAGE (STATUS "Linux x86_64 Detected")
-    ELSEIF (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "ppc64")
-        MESSAGE (STATUS "Linux ppc64 Detected")
-        SET (PROJECT_PROC_64BIT TRUE BOOL INTERNAL)
-    ENDIF (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
-ENDIF (PROJECT_OS_LINUX)
